WcaOnRails/app/models/competition.rb

```diff
@@ -29,6 +29,8 @@ class Competition < ActiveRecord::Base
            allow_nil: true,
            with_model_currency: :currency_code
 
+  scope :present, -> { where("CAST(CONCAT(endYear,'-',endMonth,'-',endDay) as Datetime) >= ?", Date.today) }
```

Comment:
`present` is actually the name we've choosen to be displayed on the competitions index page, it includes ongoing and upcoming ones. It's certainly not perfect but there hasn't been any better option so far.